### PR TITLE
Add info about K8s backup

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes

- https://docs.docker.com/ee/admin/backup/back-up-ucp/#kubernetes-settings-data-and-state
- Add more info about K8s objects included in a UCP backup

### Related issues (optional)
https://github.com/docker/docker.github.io/issues/7757

### CODE REVIEW

**Comment by traci-morrison** [comment]
@david-yu @KoryKessel-Docker @DawnWood-Docker PTAL

**Comment by traci-morrison** [comment]
@KoryKessel-Docker PTAL.

File: ee/admin/backup/back-up-ucp.md [hunk 1]
```diff
@@ -44,10 +46,12 @@ Backup contents are stored in a `.tar` file. Backups contain UCP configuration m
...
 
-UCP backups include all kubernetes declarative objects (pods, deployments, replicasets, configs...), including secrets.
+UCP backups include all Kubernetes declarative objects (pods, deployments, replicasets, configurations, and so on), including secrets. These objects are stored in the `ucp-kv etcd` database that is backed up (and restored) as part of UCP backup/restore.
 
-> **Note**: Kube volumes and kube node labels are not be backed up. 
-Upon restore, kubernetes declarative objects are re-created. Containers are re-created and IPs are resolved. 
+
+> Note
+>
+> Kubernetes volumes and node labels are not able to be backed up. Instead, upon restore, Kubernetes declarative objects are re-created. Containers are re-created and IP addresses are resolved.
```
Comment (usha-mandya) [inline ee/admin/backup/back-up-ucp.md]: Can rewrite this is active voice for clarity? Something on the lines of:

You cannot back up Kubernetes volumes and node labels.

**Comment by traci-morrison** [comment]
> LGTM with a minor change suggested above.

Thanks @usha-mandya. I incorporated your change. Can you review and approve?

**Pull request merged**